_dev/src/utils/SSCFilters.ts

```diff
@@ -119,6 +119,9 @@ export function returnCountProducts(source: Dimension): number {
   if (source.numberOfProductsAssociated !== undefined) {
     return total + Number(source.numberOfProductsAssociated);
   }
+  if (source.numberOfProductsAssociated === undefined) {
+    return total + 1;
+  }
   return total;
```

Comment:
From my understanding this is equivalent to a if / else, the `return total;` seems to be never called from now.

```suggestion
  return total + 1;
```